about.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name = "viewport" content="width=device-width, initial-scale=1">
+    <title>My Recipes</title>
+    <link rel="stylesheet" href="about.css">
+</head>
+
+<body>
+
+    <ul class="menubtn">
+        <li class="close_btn"><img src="images/close-btn.svg" alt="close-btn"></li>
+        <li class="menu-item"><a href="./index.html" class="nav-links">About</a></li>
+        <li class="menu-item"><a href="./index.html#main" class="nav-links">Recipes</a></li>
+        <li class="menu-item"><a href="./about.html#head" class="nav-links">Contact</a></li>
+    </ul>
+    <nav class="navmenu">
+        <div class="nav-title">
+            <ul class="navlist">
+                <li><img src="images/twitter.png" alt="icon"></li>
+                <li><img src="images/facebook.png" alt="icon"></li>
+                <li><a href="#"> English</a></li>
+                <li><a href="./index.html"> MyPage</a></li>
+                <li><a href="#Contact"> Logout</a></li>
+            </ul>
+        </div>
+        <nav class="nav-bar">
+        <img src="images/christmas_2012_new_2857.jpg" alt="logo" class="logoicon">
+        <ul class="navitems">
+            <li><a href="./index.html#header" class="a-abt"> Home</a></li>
+            <li><a href="#"> Courses</a></li>
+            <li><a href="./index.html#main"> Recipes</a></li>
+            <li><a href="#Logo"> Favourites</a></li>
+            <li><a href="#head"> Contact</a></li>
+        </ul>
+        <ul class="linkbox">
+            <li><a href="#Contact"> MyRecipes</a></li>
+        </ul>
+        </nav>
+    </nav>
+    <header class="abt_heading" id="head">
+        <div class="header_bg">
+            <div class="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+            <div class="info-page">
+                <h2 class="abt-hello">"Hello! myFoodies"</h2>
+                <h1 class="abt-title">RECIPES TRIED BY IPTY</h1>
+                <div class="des-border">
+                <p class="abt-descrip">I would like to share recipes that I have tried making. I love making different kinds of food especially desserts and sharing with you all is amazing. </p>
+                </div>
+                <p class="abt-contact">Please contact me per Email for any further questions.</p>
+                <a href="#">mukadamiptisam@gmail.com</a> 
+            </div>
+        </div>           
+    </header>
+    <section class="main-about">
+        <h2 class="abt-logo">My Recipes</h2>
+        <img src="images/line.png" alt="line" class="line">
+        <p class="logo-info">The logo of My recipes was decided by myself.</p>
+        <div class="myLogo">
+            <img src="images/christmas_2012_new_2857.jpg" alt="logo" class="Logoimg">
+        </div>
```

Comment:
Please, adjust the design of the images of past summits gallery to match the design.

Your design:
![image](https://user-images.githubusercontent.com/20554583/190858727-18e52a50-b190-4215-aa50-d6ad33a2f8de.png)

Design guideline:
![image](https://user-images.githubusercontent.com/20554583/190858751-8afc0e88-f95b-4dca-809f-f854ac18a8b7.png)